packages/framework/src/Resources/config/services.yml

```diff
@@ -997,3 +997,12 @@ services:
     Shopsys\FrameworkBundle\Component\Elasticsearch\ElasticsearchStructureUpdateChecker: ~
 
     Shopsys\FrameworkBundle\Component\ArrayUtils\RecursiveArraySorter: ~
+
+    Shopsys\FrameworkBundle\Component\ClassExtension\AnnotationsReplacementsMap:
+        arguments: ["@=service('Shopsys\\\\FrameworkBundle\\\\Component\\\\ClassExtension\\\\ClassExtensionRegistry').getClassExtensionMap()"]
+
+    Shopsys\FrameworkBundle\Component\ClassExtension\AnnotationsReplacer: ~
+
+    Shopsys\FrameworkBundle\Component\ClassExtension\AnnotationsAdder: ~
+
+    Shopsys\FrameworkBundle\Component\ClassExtension\FileContentsReplacer: ~
```

Comment:
new line please (we need a fixer for that 😄 )

Reply:
fixed in 9e0d477dc. Could you please create an issue or PR for that? :grin:

Reply:
Created #1367